fairer-iOS/fairer-iOS/Screens/Statistics/RankSectionHeaderReusableView.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  RankSectionHeaderReusableView.swift
+//  fairer-iOS
+//
+//  Created by 김규철 on 2023/08/30.
+//
+
+import UIKit
+
+final class RankSectionHeaderReusableView: UICollectionReusableView {
+    
+    private let titleLabel: UILabel = {
+        let label = UILabel()
+        label.text = "이번달 김유나님이\n가장 많은 집안일을 완료했어요!"
+        label.textColor = .black
+        label.textAlignment = .left
+        label.numberOfLines = 2
+        label.font = .h2
+        return label
+    }()
```

Comment:
```suggestion
        private let titleLabel: UILabel = {
        let label = UILabel()
        label.setTextWithLineHeight(text: "이번달 김유나님이\n가장 많은 집안일을 완료했어요!", lineHeight: 22)
        label.textColor = .gray800
        label.textAlignment = .left
        label.numberOfLines = 2
        label.font = .title1
        return label
    }()
```
title1, gray800 으로 수정해주시고
text 대신 `setTextWithLineHeight(text: "이번달 김유나님이\n가장 많은 집안일을 완료했어요!", lineHeight: 22)` 으로
lineheight 적용 부탁드려요~